tools/bin/commands/util/operator_funcs

```diff
@@ -88,17 +88,26 @@ download_operator_binary() {
     esac
 
     if [[ "${pattern}" != "unknown" ]]; then
-        release_uri="https://api.github.com/repos/syndesisio/syndesis/releases?per_page=1"
         release_tag="$(readopt --tag)"
+        url=""
         if [[ -n "$release_tag" ]]; then
             release_uri="https://api.github.com/repos/syndesisio/syndesis/releases/tags/${release_tag}"
+            url=$(curl -s $release_uri \
+                | grep browser_download_url \
+                | grep ${pattern} \
+                | cut -d '"' -f 4)
+        else
+            # retrieve the last 4 releases and get the latest 2.0 release
+            url=$(curl -s "https://api.github.com/repos/syndesisio/syndesis/releases?per_page=4" \
+                | grep "download/2" \
```

Comment:
This will blow up on us when we switch to 3.x.y, not sure if anyone will remember this change at that point...

Reply:
When it downloads syndesis-operator it prints the exact version, so we are going to see it, at least. Currently it downloads the last tag, which was 1.9 yesterday, today is 2.0, unpredictable. Later we may add an option to specify the major version which we want to download.